gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -1181,7 +1192,7 @@ gnc_plugin_page_register_create_widget (GncPluginPage *plugin_page)
         g_free(order);
 
         priv->sd.reverse_order = gnc_plugin_page_register_get_sort_reversed(plugin_page);
-        gnc_split_reg_set_sort_reversed(priv->gsr, priv->sd.reverse_order, FALSE);
+        gnc_split_reg_set_sort_reversed(priv->gsr, priv->sd.reverse_order, priv->enable_refresh);
```

Comment:
priv->enable_refresh is always FALSE here, so it's confusing to change it.

Reply:
I think I got carried away with the find and replace changes.